dask/dataframe/core.py

```diff
@@ -351,6 +353,57 @@ def __init__(self, dsk, name, meta, divisions):
         self._meta = meta
         self.divisions = tuple(divisions)
 
+        # Optionally cast object dtypes to `pyarrow` strings
+        if dask.config.get("dataframe.convert_string"):
+            try:
+                import pyarrow  # noqa: F401
+            except ImportError:
+                raise RuntimeError(
+                    "Using dask's `dataframe.convert_string` configuration "
+                    "option requires `pyarrow` to be installed."
```

Comment:
Any specific version of pyarrow?

Reply:
I think 1.0. At least that what `pandas=1.3` lists as the minimum supported version.